plone/app/content/testing.py

```diff
@@ -88,14 +127,16 @@ def setUpPloneSite(self, portal):
     bases=(PLONE_APP_CONTENT_FIXTURE, ),
     name="PloneAppContent:Functional")
 
+
 # Dexterity test layers
 PLONE_APP_CONTENT_DX_INTEGRATION_TESTING = IntegrationTesting(
-    bases=(PLONE_APP_CONTENTTYPES_FIXTURE, ),
+    bases=(PLONE_APP_CONTENT_BASE_FIXTURE, PLONE_APP_CONTENTTYPES_FIXTURE, ),
     name="PloneAppContentDX:Integration")
 PLONE_APP_CONTENT_DX_FUNCTIONAL_TESTING = FunctionalTesting(
-    bases=(PLONE_APP_CONTENTTYPES_FIXTURE, ),
+    bases=(PLONE_APP_CONTENT_BASE_FIXTURE, PLONE_APP_CONTENTTYPES_FIXTURE, ),
```

Comment:
I'm using this test layer to fix the test isolation problem. The ``PLONE_APP_CONTENT_BASE_FIXTURE`` is new and added. It might not be necessary for the other tests depending on these test layers, but IMO it makes sense to setup plone.app.content in any case for tests from plone.app.content, no?